### ISSUE TITLE
can instapy support proxy and VPN?

### ISSUE THREAD

**Issue by gitxxxa** [InstaPy/InstaPy#6431]
## Expected Behavior
support proxy and VPN
## Current Behavior
error reported when using http proxy
## InstaPy configuration
default configruation

Can instapy support proxy and vpn?  someone can help on this?

**Comment by CodeMasterBond** [comment]
Yes, instapy does support proxies and you can use a VPN with it. There is literally a file named for it.....
https://github.com/InstaPy/InstaPy/blob/master/proxy_extension.py
